model/repository.go

```diff
@@ -24,14 +24,12 @@ type Repository interface {
 }
 
 type EntRepository struct {
-	client  *ent.Client
-	storage storagePkg.Storage
+	client *ent.Client
 }
 
 func NewEntRepository(client *ent.Client, storage storagePkg.Storage) *EntRepository {
```

Comment:
引数の`storage`を使っていないので、消して欲しいです(`NewEntRepository`を使っているところも合わせる必要があります)

```suggestion
func NewEntRepository(client *ent.Client) *EntRepository {
```